cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -387,7 +387,7 @@ func NewControllerInitializers(loopMode ControllerLoopMode) map[string]InitFunc
 	if loopMode == IncludeCloudLoops {
 		controllers["service"] = startServiceController
 		controllers["route"] = startRouteController
-		controllers["cloudnodelifecycle"] = startCloudNodeLifecycleController
+		controllers["cloud-node-lifecycle"] = startCloudNodeLifecycleController
```

Comment:
For KCM reviewer: this changed was merged in https://github.com/kubernetes/kubernetes/pull/70344 (i.e. in v1.14) so it is not breaking compatibility for anyone